### DISCUSSION TITLE
Which version

### DISCUSSION THREAD

**Discussion by ljungis** [magma/magma#14991] [Q&A] [answered]
Hi, starting virtualbox I am prompted to update. But I'm not shore about which version to choose. 
Thanksful for any help.

![Skärmbild från 2023-02-10 11-50-34](https://user-images.githubusercontent.com/37312627/218077203-cc9732d6-90d0-406e-af50-3dbc6e2a3f6e.png)
![Skärmbild från 2023-02-10 11-50-52](https://user-images.githubusercontent.com/37312627/218077211-28e2071d-d5fc-4121-8320-7a09e1092802.png)
![Skärmbild från 2023-02-10 11-52-23](https://user-images.githubusercontent.com/37312627/218077217-21acf9ab-d0ab-4cee-80f6-4f510c65f10c.png)

**Comment by LKreutzer** [comment]
Hi @ljungis , a 6.X version will work well - I can recommend the 6.1.40 version for Virtualbox. We have seen some issues using the newest 7.X versions.

**Reply by LKreutzer** [reply]
If this solves your issue, please mark this as the answer, so that the discussion is shown as resolved.

**Comment by ljungis** [comment]
Thank's for your quick reply.
6.1.40 version is what I am using.
So there is no need for updating.
But is there a way to disable the update prompt?

**Reply by LKreutzer** [reply]
That is right, there is currently no need to update. 

You can configure the update settings for VirtualBox by going to `File` -> `Preferences` -> `Update`:
![Screenshot from 2023-02-10 13-05-20](https://user-images.githubusercontent.com/34488763/218088545-8c4e3561-0a16-49b9-b2ad-4411fb433fe8.png)

**Comment by ljungis** [comment]
Thank's for your quick reply.
6.1.40 version is what I am using.
So there is no need for updating.
But is there a way to disable the update prompt?

Den fre 10 feb. 2023 kl 12:52 skrev Lars Kreutzer ***@***.***

**Comment by ljungis** [comment, marked as answer]
Thankś for the help.
